frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -46,9 +41,9 @@ const resourceAttributes = (model: K8sKind, namespace: string): AccessReviewReso
 
 const ODCEmptyState: React.FC<Props> = ({
   title,
-  activeNamespace,
   hintBlock = 'Select a way to create an application, component or service from one of the options.',
 }) => {
+  const activeNamespace = getActiveNamespace();
```

Comment:
Why are using `getActiveNamespace()` here which directly hooks into the redux store variable and gives back the result? This is not a correct convention to use redux state and console code does it wrong. We should always use `connect` to get the values from redux state.

Reply:
The old code was correct. Please revert changes to this file.